master/buildbot/db/users.py

```diff
@@ -28,21 +28,23 @@ class UsDict(dict):
 class UsersConnectorComponent(base.DBConnectorComponent):
     # Documentation is in developer/db.rst
 
+    def __init__(self, *args, **kwargs):
+        super(UsersConnectorComponent, self).__init__(*args, **kwargs)
+        self.tbl = self.db.model.users
```

Comment:
are these two part of the API?  if not, make them private (prepend the names with `_`).